lua/starfall/libs_sh/input.lua

```diff
@@ -165,6 +165,21 @@ function wpanel:OnMouseWheeled(delta)
 	end
 end
 
+--- Called when a keybind is pressed
+-- @client
+-- @name playerBindPressed
+-- @class hook
+-- @param Player ply Player pressing the keybind
+-- @param string bind Name of keybind pressed
+-- @param boolean pressed If the key is pressed. This will always return true as it is a GMod bug.
```

Comment:
You can remove the 'pressed' arg since it doesn't work.

Also could rename playerbindpressed to inputbindpressed so its with starfall's inputpressed.

I'd also move this to input.lua after where inputpressed is defined.

Reply:
Added these suggestions